app/controllers/genotypes_controller.rb

```diff
@@ -47,7 +47,7 @@ def show
 
   def feed
     # for rss-feeds
-    @genotypes = Genotype.all(:order => "created_at DESC", :limit => 20)
+    @genotypes = Genotype.order('created_at DESC').limit(20)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.